### PR TITLE
Feat: activities endpoint

### PR DESCRIPTION
**Pull request by PeterSmallenbroek**
[AB#30438](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/30438) 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added tests wherever relevant
- [ ] I have made sure that all automated checks pass before requesting a review
- [ ] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: services/121-service/src/scoped.repository.ts
Comment (diderikvw) [inline services/121-service/src/scoped.repository.ts:1]: I do not understand what is going on here.

File: services/121-service/src/utils/scope/createScopedRepositoryProvider.helper.ts
Comment (diderikvw) [inline services/121-service/src/utils/scope/createScopedRepositoryProvider.helper.ts:1]: I do not understand what is going on here.

**Review by diderikvw — COMMENTED** [review]
A lot of great work to build on!

File: services/121-service/src/scoped.repository.ts
Comment (PeterSmallenbroek) [reply services/121-service/src/scoped.repository.ts:1]: This was copied from the Visa branch to make the scoped repositories work.

File: services/121-service/src/utils/scope/createScopedRepositoryProvider.helper.ts
Comment (PeterSmallenbroek) [reply services/121-service/src/utils/scope/createScopedRepositoryProvider.helper.ts:1]: This was copied from the Visa branch to make the scoped repositories work.

**Review by aberonni — APPROVED** [review]
Approving this based on the sub-branches already being reviewed

**Pull request merged**